packages/data-stores/src/analyzer/types.ts

```diff
@@ -0,0 +1,31 @@
+import * as actions from './actions';
+import type { DispatchFromMap } from '../mapped-types';
+
+interface Color {
+	name: string;
+	hex: string;
+}
+
+export interface ColorsData {
+	logo: Color[];
+	links: Color[];
+	favicon: Color[];
+	body?: Color;
+	theme_color?: Color;
+}
+
+export interface AnalyzeColorsResponse {
+	url: string;
+	status: string;
+	colors: ColorsData;
```

Comment:
I think that these types should be updated to last [url-analyzer](https://github.com/Automattic/importer-functions/blob/master/functions/url-analyzer/src/types.ts#L47-L84) types. The structure that you find in production right now has been changed. Isn't this true @jeroenpf?

Reply:
You're right. This type definition is outdated.

Reply:
Updated.